### DISCUSSION TITLE
Specify multiple interfaces to listen on

### DISCUSSION THREAD

**Discussion by shockogit** [genotrance/px#179] [Q&A]
I have several interfaces each with a single IP address on my system. Is it possible to use multiple listen directives via the CLI or .ini file? It seems from testing we can only specify one or all (0.0.0.0). I need the proxy to listening on only 2 interfaces. I can of course spin up two instances but I'd prefer one!

**Comment by genotrance** [comment]
This has been implemented in #195.
